lib/sign_in/widgets/form.dart

```diff
@@ -1,10 +1,13 @@
 import 'package:black_hole_flutter/black_hole_flutter.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:pedantic/pedantic.dart';
 import 'package:schulcloud/app/app.dart';
+import 'package:schulcloud/sign_in/widgets/signin_browser.dart';
 import 'package:webview_flutter/webview_flutter.dart';
 
+import '../../app/app.dart';
```

Comment:
Please use absolute imports for files of different modules. Also, the app module is already imported in line 6